pysages/methods/abf.py

```diff
@@ -1,48 +1,129 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 # Copyright (c) 2020-2021: PySAGES contributors
 # See LICENSE.md and CONTRIBUTORS.md at https://github.com/SSAGESLabs/PySAGES
+"""
+Adaptive Biasing Force (ABF) sampling method.
 
-from collections import namedtuple
+ABF partitions the collective variable space into bins determined by a user
+provided grid, and keeping a tabulation of the number of visits to each bin
+as well as the sum of generalized forces experienced by the system at each
+configuration bin. These provide an estimate for the mean generalized force,
+which can be integrated to yield the free energy.
 
-from jax import scipy
+The implementation of the adaptive biasing force method here closely follows
+https://doi.org/10.1063/1.2829861. One important difference is that the time
+derivative of the product W·p (equation 9 of reference) is approximated by as
```

Comment:
I have mathjax support enabled.
If you want, you can use it to write this as a math formula here.
https://www.sphinx-doc.org/en/1.5.1/ext/math.html